### PR TITLE
[Feature] Add RTMW3D for 3D wholebody pose estimation task

### PR DESCRIPTION
**Pull request by xiexinch**
## Motivation

Add RTMPose3D model.

## Modification

- Add rtmpose3d to projects
- Minor change to data pipeline and 3d datasets

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/datasets/datasets/wholebody3d/h3wb_dataset.py [hunk 1]
```diff
@@ -163,6 +171,20 @@ def _load_annotations(self) -> Tuple[List[dict], List[dict]]:
...
+                            bbox['x_min'], bbox['y_min'], bbox['x_max'],
+                            bbox['y_max']
+                        ]],
+                                        dtype=np.float32)
+
+                        img_path = f'{self.data_root}original/{subject}/Images/{act}.{cam}/frame_{frames[frame_ids[-1]]}.jpg'  # noqa
+                        img_paths = [
+                            f'{self.data_root}original/{subject}/Images/{act}.{cam}/frame_{frames[i]}.jpg'  # noqa
+                            for i in frame_ids
+                        ]
```
Comment (Tau-J) [inline mmpose/datasets/datasets/wholebody3d/h3wb_dataset.py:186]: I suggest using `osp.join()` to improve robustness.

Comment (Tau-J) [reply mmpose/datasets/datasets/wholebody3d/h3wb_dataset.py:186]: In case there is no `'/'` in `self.data_root`

**Comment by stihuangyuan** [comment]
rtmpose3d train dataset, h3wb_dataset h3wb_train_bbox.npz ann file could not find!

**Comment by zcl912** [comment]
> rtmpose3d train dataset, h3wb_dataset h3wb_train_bbox.npz ann file could not find!
hello, have you ever find the h3wb_train_bbox.npz file

**Pull request merged**